worlds/TheLocalPub/Prefabs/Characters/Factions/OPFOR/USSR_Army/Extra/Character_USSR_Crewman_driver_P.et

```diff
@@ -0,0 +1,24 @@
+SCR_ChimeraCharacter : "{D2E883F5E6911DBA}Prefabs/Characters/Factions/OPFOR/USSR_Army/Character_USSR_Unarmed_P.et" {
```

Comment:
We have renamed variants of the unarmed USSR crew in GC units if you wanted to use that version instead to cut out these prefabs, not a neccessity though